.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: test
+run-name: ${{ github.actor }} runs test
+on:
+  pull_request:
+    types: [opened]
+jobs:
+  check-bats-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.12.x'
```

Comment:
The code is unnecessarily complex. You could simplify it by using the `poetry add` command directly instead of installing poetry separately.